src/main/java/teammates/ui/template/FeedbackSessionsForm.java

```diff
@@ -158,6 +158,28 @@ public static FeedbackSessionsForm getFormForNewFs(FeedbackSessionAttributes fee
         return newFsForm;
     }
 
+    public String getInstructorSubmissionStatus(FeedbackSessionAttributes session) {
```

Comment:
Why not just make this (and the other status) fields of `FeedbackSessionsForm`? That way the JSP file can get the value with just a simple getter.